Formula/geode.rb

```diff
@@ -0,0 +1,36 @@
+class Geode < Formula
+  desc "In-memory Data Grid for fast transactional data processing"
+  homepage "https://geode.apache.org"
+  url "http://apache.org/dyn/closer.cgi?filename=incubator/geode/1.0.0-incubating.M2/apache-geode-1.0.0-incubating.M2.tar.gz&action=download"
+  version "1.0.0-incubating.M2"
```

Comment:
Off the top of my head some Apache stuff do releases that changes M2 to M3 or such, with no other version change. Could be misremembering though.